tweaks/steam.yaml

```diff
@@ -377,9 +377,6 @@
 "606280": # Darksiders III
   compat_tool: proton_5
 
-"384310": # Dead In Bermuda
```

Comment:
Nice catch, but we should probably delete the other one since this one is in correct alphabetical order.

Reply:
Got it, I'll swap them

Reply:
Fixed